### PR TITLE
Alarmdecoder doc updates for new zone loop parameter

### PR DESCRIPTION
**Pull request by jkeljo**
**Description:**
Docs for new config option.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#18477

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/alarmdecoder.markdown [hunk 1]
```diff
@@ -40,6 +40,7 @@ alarmdecoder:
       name: 'Smoke Detector'
       type: 'smoke'
       rfid: '0123456'
+      loop: 1
```
Comment (frenck) [inline source/_components/alarmdecoder.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Just a small note there @jkeljo 👍 
I know there are more optional configuration options in the example, but we should not make it worse.

File: source/_components/alarmdecoder.markdown (hunk 1 above)
Comment (jkeljo) [reply source/_components/alarmdecoder.markdown]: No problem. :-)

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**